src/svd/registerinfo.rs

```diff
@@ -105,15 +105,22 @@ impl Encode for RegisterInfo {
             attributes: HashMap::new(),
             children: vec![
                 new_element("name", Some(self.name.clone())),
-                new_element("description", Some(self.description.clone())),
                 new_element(
                     "addressOffset",
                     Some(format!("0x{:x}", self.address_offset)),
                 ),
             ],
             text: None,
         };
-
+        match self.description {
+            Some(ref v) => {
+                elem.children.push(new_element(
```

Comment:
This will fail ci. This is because the encode tests are dependent on the order of the children vec. To fix this we can either make the test order agnostic with a sort on all element children or use an insert.
I'm not sure which one I prefer, as making the test order agnostic seems like a more permanent solution should this issue arise again

```suggestion
                elem.children.insert(1, new_element(
```

Reply:
Order agnostic sounds good to mee!